master/internal/api_workspace_intg_test.go

```diff
@@ -521,3 +522,29 @@ func TestAuthzWorkspaceGetThenActionRoutes(t *testing.T) {
 		require.Equal(t, expectedErr.Error(), curCase.IDToReqCall(id).Error())
 	}
 }
+
+func TestWorkspaceHasModels(t *testing.T) {
+	// set up a dB and api server to use for integration testing
+	require.NoError(t, etc.SetRootPath("../static/srv"))
+	pgDB, cleanup := db.MustResolveNewPostgresDatabase(t)
+	defer cleanup()
+	db.MustMigrateTestPostgres(t, pgDB, "file://../static/migrations")
+	api, _, ctx := setupAPITest(t, pgDB)
+
+	// create workspace for test
+	workspaceName := "test-workspace"
+	resp, err := api.PostWorkspace(ctx, &apiv1.PostWorkspaceRequest{Name: workspaceName})
+	require.NoError(t, err)
+
+	// confirm workspace does not have any models
+	exists, err := api.workspaceHasModels(ctx, resp.Workspace.Id)
+	assert.False(t, exists)
+	require.NoError(t, err)
```

Comment:
You should always try to check an error before looking at other values a function returns. So I would switch the order of these statements.